src/zenml/client.py

```diff
@@ -4435,23 +4435,17 @@ def _delete_artifact_from_artifact_store(
     def create_run_metadata(
         self,
         metadata: Dict[str, "MetadataType"],
-        resource_id: UUID,
-        resource_type: MetadataResourceTypes,
+        resources: List[Tuple[UUID, MetadataResourceTypes]],
```

Comment:
Maybe it deserves a proper class since this is user-facing and it is usually quite hard to understand those Tuple args, at least for someone like me.

Reply:
I also thought about this, and I was not so sure about which path to follow. On the one hand, it makes the code easier to read and follow, but on the other hand, it feels redundant to add a new class just to combine an ID and a type for an internal use case. I will try it out to see how it fits.

Reply:
I guess, in this case, it is not fully internal as it is a `client` method, which makes it even worse to add a new class IMO.

Reply:
Well, you can make it a private class method, if it is meant to be only used internally. In this case, it should be fine to keep awkward input parameters, since the "we told you not to use it" thing.